app/components/draft_scheduled_post/draft_scheduled_post.tsx

```diff
@@ -92,33 +108,41 @@ const Draft: React.FC<Props> = ({
             underlayColor={changeOpacity(theme.centerChannelColor, 0.1)}
             testID='draft_post'
         >
-            <View
-                style={style.container}
-            >
-                <DraftPostHeader
-                    channel={channel}
-                    draftReceiverUser={draftReceiverUser}
-                    rootId={draft.rootId}
-                    testID='draft_post.channel_info'
-                    updateAt={draft.updateAt}
-                />
-                {showPostPriority && draft.metadata?.priority &&
-                <View style={style.postPriority}>
-                    <Header
-                        noMentionsError={false}
-                        postPriority={draft.metadata?.priority}
+            <View style={style.container}>
+                {draftType === DRAFT_TYPE_SCHEDULED && (post as ScheduledPostModel).errorCode !== '' &&
+                    <View style={style.errorLine}/>
+                }
```

Comment:
0/5 I don't see any testes for this new behavior.

Reply:
Same